src/com/projectkorra/projectkorra/firebending/combustion/Combustion.java

```diff
@@ -115,19 +116,16 @@ private void advanceLocation() {
 	}
 
 	private void createExplosion(final Location block, final float power, final boolean canBreakBlocks) {
-		ParticleEffect.EXPLOSION_LARGE.display(block, 3, 2, 2, 2, 0);
-		
-		if (canFireGrief()) {
-			block.getWorld().createExplosion(block.getX(), block.getY(), block.getZ(), power, true, canBreakBlocks);
-		}
-		for (final Entity entity : block.getWorld().getEntities()) {
+		for (Entity entity : GeneralMethods.getEntitiesAroundPoint(block, power)) {
 			if (entity instanceof LivingEntity) {
 				if (entity.getLocation().distanceSquared(block) < this.radius * this.radius) { // They are close enough to the explosion.
 					DamageHandler.damageEntity(entity, this.damage, this);
 					AirAbility.breakBreathbendingHold(entity);
 				}
 			}
 		}
+
+		block.getWorld().createExplosion(block.getX(), block.getY(), block.getZ(), power, canFireGrief(), canBreakBlocks);

```

Comment:
Where is ParticleEffect.EXPLOSION_LARGE.display(block, 3, 2, 2, 2, 0); now?
Did you just remove these particles?

Reply:
They are no longer needed since the explosion is now always called. And the explosion itself has the particles